Recruit-Training/Embedded-Recruit-Training/Core/Src/BlueLedToggleTask.c

```diff
@@ -9,8 +12,39 @@ void blueLedToggleTask(void const* arg)
 
     for (;;)
     {
-        osDelayUntil(prevWakeTime + 1);  //TODO: Replace 1 with time period for delay
+        osDelayUntil(prevWakeTime + BLUE_LED_TOGGLE_FREQ);  //TODO: Replace 1 with time period for delay
+        
         //TODO: Check blue toggle flag and toggle blue LED
+        if (blueToggleFlag)
+        {
+            HAL_GPIO_TogglePin(GPIOA, GPIO_PIN_5);
```

Comment:
You can find the user friendly names for these in main.h (you can use LED_RED for the port and pin since we forgot to add the blue LED haha)